### ISSUE TITLE
 Image information with a multilevel path in the name is not displayed

### ISSUE THREAD

**Issue by GSergeevich** [jc21/docker-registry-ui#21]
If the name is like a "tcmp/pr/billing-fork" , image information page is empty

**Comment by cgrard** [comment]
Yes, same here, all our images are tagged with this kind of pattern which makes the UI basically useless :/

**Comment by rlogiacco** [comment]
Same here, we use `prj/project_name/prj_module` and it fails

**Comment by rlogiacco** [comment]
I believe this is a duplicate of #10

**Comment by rlogiacco** [comment]
After a little investigation, the issue should be in `src/frontend/js/routes/image.js` as calling the API directly works nicely, but it seems the UI (frontend) is not calling the API (backend)... probably due to some failing check.

 I might be able to provide additional info if we get a non-minified version.
As it looks the author is not interested in moving this forward, is anyone having this issue willing to pair up on this?

**Comment by cgrard** [comment]
Yep, I'm not really familiar with the stuff that has been used to build the frontend but I can try.
